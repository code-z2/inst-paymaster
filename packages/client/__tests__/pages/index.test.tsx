import React from "react";
import { render, screen } from "@testing-library/react";
import "@testing-library/jest-dom";
import Landingpage from "../../pages";
import SecureLandingPage from "../../pages/auth/[private]";
import SecurePaymasterListPage from "../../pages/auth/[private]/activepaymaster";

/*
 *  snapshots of the index page
 */
describe("Landing page ", () => {
  it("should render the landing page ", () => {
    const home = render(<Landingpage />);
    expect(home).toMatchSnapshot();
  });
});
