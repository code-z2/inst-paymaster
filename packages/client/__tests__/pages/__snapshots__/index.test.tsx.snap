// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Landing page  should render the landing page  1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="container scaleto100 mx-auto min-h-(65vh)"
      >
        <div
          class="grid-container"
        >
          <nav
            class="paynav"
          >
            <a
              class="paylogo"
              href="/"
            >
              <img
                alt="paymaster logo"
                data-nimg="1"
                decoding="async"
                height="50"
                src="/_next/image?url=%2Fassets%2Fpm-images%2Flogo.png&w=256&q=75"
                srcset="/_next/image?url=%2Fassets%2Fpm-images%2Flogo.png&w=128&q=75 1x, /_next/image?url=%2Fassets%2Fpm-images%2Flogo.png&w=256&q=75 2x"
                style="color: transparent;"
                width="120"
              />
            </a>
            <ul
              class="list-ul"
            >
              <li
                data-testid="list-items"
              >
                <a
                  class="mx-1 py-1"
                  href="#"
                >
                  Docs
                </a>
              </li>
              <li
                data-testid="list-items"
              >
                <button
                  class="px-1 m-0 py-1 lg:w-40 hover:bg-purple-600 border-solid border-2 border-slate-400 rounded-md"
                >
                  Coming Soon
                </button>
              </li>
            </ul>
          </nav>
          <section
            class="first_section"
          >
            <div
              class="heading_section"
            >
              <h1>
                The only paymaster's access infrastructure you will need
              </h1>
              <p
                class="text-slate-400 my-6"
              >
                Using paymaster to simplify zk transactions and provide
                <br />
                 superior ux to dapps
              </p>
              <button
                class="px-3 py-1 "
              >
                Coming soon
              </button>
            </div>
            <div
              class="heading_image"
            >
              <img
                alt="paymaster-planet-logo"
                data-nimg="1"
                data-testid="planet_img"
                decoding="async"
                height="322"
                src="/_next/image?url=%2Fassets%2Fpm-images%2Fheader_image.png&w=1080&q=75"
                srcset="/_next/image?url=%2Fassets%2Fpm-images%2Fheader_image.png&w=640&q=75 1x, /_next/image?url=%2Fassets%2Fpm-images%2Fheader_image.png&w=1080&q=75 2x"
                style="color: transparent;"
                width="452"
              />
            </div>
            <div
              class="planet_blur"
            />
          </section>
          <footer
            class="bottom pl-2"
          >
            <hr
              class=""
            />
            <p>
              © 2023 Paymasters. All rights reserved. 
            </p>
            <ul>
              <li
                data-testid="footer-list-item"
              >
                <a
                  href="#"
                >
                  Discord
                </a>
                 
              </li>
              <li
                data-testid="footer-list-item"
              >
                <a
                  href="#"
                >
                  Telegram 
                </a>
              </li>
              <li
                data-testid="footer-list-item"
              >
                <a
                  href="#"
                >
                  Twitter 
                </a>
              </li>
              <li
                data-testid="footer-list-item"
              >
                <a
                  href="#"
                >
                  Github
                </a>
                 
              </li>
              <li
                data-testid="footer-list-item"
              >
                <a
                  href="#"
                >
                  Contact
                </a>
                 
              </li>
              <li
                data-testid="footer-list-item"
              >
                <a
                  href="#"
                >
                  Privacy
                </a>
                 
              </li>
            </ul>
          </footer>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="container scaleto100 mx-auto min-h-(65vh)"
    >
      <div
        class="grid-container"
      >
        <nav
          class="paynav"
        >
          <a
            class="paylogo"
            href="/"
          >
            <img
              alt="paymaster logo"
              data-nimg="1"
              decoding="async"
              height="50"
              src="/_next/image?url=%2Fassets%2Fpm-images%2Flogo.png&w=256&q=75"
              srcset="/_next/image?url=%2Fassets%2Fpm-images%2Flogo.png&w=128&q=75 1x, /_next/image?url=%2Fassets%2Fpm-images%2Flogo.png&w=256&q=75 2x"
              style="color: transparent;"
              width="120"
            />
          </a>
          <ul
            class="list-ul"
          >
            <li
              data-testid="list-items"
            >
              <a
                class="mx-1 py-1"
                href="#"
              >
                Docs
              </a>
            </li>
            <li
              data-testid="list-items"
            >
              <button
                class="px-1 m-0 py-1 lg:w-40 hover:bg-purple-600 border-solid border-2 border-slate-400 rounded-md"
              >
                Coming Soon
              </button>
            </li>
          </ul>
        </nav>
        <section
          class="first_section"
        >
          <div
            class="heading_section"
          >
            <h1>
              The only paymaster's access infrastructure you will need
            </h1>
            <p
              class="text-slate-400 my-6"
            >
              Using paymaster to simplify zk transactions and provide
              <br />
               superior ux to dapps
            </p>
            <button
              class="px-3 py-1 "
            >
              Coming soon
            </button>
          </div>
          <div
            class="heading_image"
          >
            <img
              alt="paymaster-planet-logo"
              data-nimg="1"
              data-testid="planet_img"
              decoding="async"
              height="322"
              src="/_next/image?url=%2Fassets%2Fpm-images%2Fheader_image.png&w=1080&q=75"
              srcset="/_next/image?url=%2Fassets%2Fpm-images%2Fheader_image.png&w=640&q=75 1x, /_next/image?url=%2Fassets%2Fpm-images%2Fheader_image.png&w=1080&q=75 2x"
              style="color: transparent;"
              width="452"
            />
          </div>
          <div
            class="planet_blur"
          />
        </section>
        <footer
          class="bottom pl-2"
        >
          <hr
            class=""
          />
          <p>
            © 2023 Paymasters. All rights reserved. 
          </p>
          <ul>
            <li
              data-testid="footer-list-item"
            >
              <a
                href="#"
              >
                Discord
              </a>
               
            </li>
            <li
              data-testid="footer-list-item"
            >
              <a
                href="#"
              >
                Telegram 
              </a>
            </li>
            <li
              data-testid="footer-list-item"
            >
              <a
                href="#"
              >
                Twitter 
              </a>
            </li>
            <li
              data-testid="footer-list-item"
            >
              <a
                href="#"
              >
                Github
              </a>
               
            </li>
            <li
              data-testid="footer-list-item"
            >
              <a
                href="#"
              >
                Contact
              </a>
               
            </li>
            <li
              data-testid="footer-list-item"
            >
              <a
                href="#"
              >
                Privacy
              </a>
               
            </li>
          </ul>
        </footer>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
